packages/client/src/comp/states/Base.tsx

```diff
@@ -84,6 +87,10 @@ const ContainerlessBase = <Info extends StateInfo>({ enrichValues, children }: P
       return null
     }
 
+    const hasMailingAddress =
```

Comment:
I think we need to pull this state out into `BaseContainer`.  This fails if a voter clicks on the toggle, types in a mailing address, and then disables the toggle again because `mailingFields` will still be used.